--- jarjar-1.0.orig/src/main/com/tonicsystems/jarjar/util/AntJarProcessor.java	2010-10-01 12:22:33.451738707 +0100
+++ jarjar-1.0/src/main/com/tonicsystems/jarjar/util/AntJarProcessor.java	2010-10-01 12:25:06.935738716 +0100
@@ -84,7 +84,7 @@
         String dir = file.substring(0, slash);
         if (dirs.add(dir)) {
           addParentDirs(dir, zOut);
-          super.zipDir(null, zOut, dir + "/", ZipFileSet.DEFAULT_DIR_MODE, JAR_MARKER);
+          super.zipDir((File)null, zOut, dir + "/", ZipFileSet.DEFAULT_DIR_MODE, JAR_MARKER);
         }
       }
     }
